Tests/UnitTests/Networking/Responses/OfferingsDecodingTests.swift

```diff
@@ -99,6 +99,33 @@ class OfferingsDecodingTests: BaseHTTPResponseTest {
         expect(package.platformProductIdentifier) == "com.revenuecat.other_product"
     }
 
+    func testDecodesPaywallData() throws {
+        let offering = try XCTUnwrap(self.response.offerings[safe: 2])
+
+        expect(offering.identifier) == "paywall"
+        expect(offering.description) == "Offering with paywall"
+        expect(offering.metadata) == [:]
+        expect(offering.packages).to(haveCount(2))
+
+        let paywall = try XCTUnwrap(offering.paywall)
+        expect(paywall.template) == .example1
+        expect(paywall.defaultLocale) == Locale(identifier: "en_us")
```

Comment:
Should this be `en_US'?

```suggestion
        expect(paywall.defaultLocale) == Locale(identifier: "en_US")
```

Reply:
Oh yeah thanks